src/PubSubHubbub/Subscriber.php

```diff
@@ -735,19 +745,22 @@ protected function _getRequestParameters($hubUrl, $mode)
             $expires = $now->add(new DateInterval('PT' . $params['hub.lease_seconds'] . 'S'))
                 ->format('Y-m-d H:i:s');
         }
+
+        // phpcs:disable Generic.Files.LineLength.TooLong
         $data = [
-            'id'              => $key,
-            'topic_url'       => $params['hub.topic'],
-            'hub_url'         => $hubUrl,
-            'created_time'    => $now->format('Y-m-d H:i:s'),
-            'lease_seconds'   => $params['hub.lease_seconds'],
-            'verify_token'    => hash('sha256', $params['hub.verify_token']),
-            'secret'          => null,
-            'expiration_time' => $expires,
-            // @codingStandardsIgnoreStart
-            'subscription_state' => ($mode == 'unsubscribe') ? PubSubHubbub::SUBSCRIPTION_TODELETE : PubSubHubbub::SUBSCRIPTION_NOTVERIFIED,
-            // @codingStandardsIgnoreEnd
+            'id'            => $key,
+            'topic_url'     => $params['hub.topic'],
+            'hub_url'       => $hubUrl,
+            'created_time'  => $now->format('Y-m-d H:i:s'),
+            'lease_seconds' => $params['hub.lease_seconds'],
+            /** @psalm-suppress PossiblyInvalidCast */
```

Comment:
Seems fishy - `$params['hub.verify_token']` always contains a `string` at this location?

Reply:
If not, a comment is needed - can't wrap my mind around this comment set here as standalone

Reply:
Specifically, this is generated by:

```php
    protected function _generateVerifyToken()
    {
        // @codingStandardsIgnoreEnd
        if (! empty($this->testStaticToken)) {
            return $this->testStaticToken;
        }
        return uniqid(rand(), true) . time();
    }
```

Reply:
The only change here was the indentation. However, Psalm saw that, and raised an error.

But yes, at this point, the value is always a string here.

Reply:
Hm, usually, psalm does not care about indentation or the line where the error occured (as phpstan does).
It only stores "PossiblyInvalidCast appears 2 times in file XY" within the baseline. 🤔